src/main/java/org/truffleruby/core/rope/RopeNodes.java

```diff
@@ -1107,4 +1111,169 @@ public LeafRope flatten(Rope rope) {
 
     }
 
+    @NodeChildren({
+            @NodeChild(type = RubyNode.class, value = "a"),
+            @NodeChild(type = RubyNode.class, value = "b")
+    })
+    public abstract static class EqualNode extends RubyNode {
```

Comment:
I think that's the only case that matters, so we could inline the method there I believe.